localstack/init-localstack-resources.sh

```diff
@@ -3,3 +3,13 @@
 awslocal s3 mb s3://drb-files-local
 awslocal s3 mb s3://drb-files-limited-local
 awslocal s3 mb s3://ump-pdf-repository-local
+
+awslocal s3api put-bucket-cors --bucket drb-files-local --cors-configuration '{
+  "CORSRules": [
+    {
+      "AllowedOrigins": ["*"],
+      "AllowedMethods": ["GET", "POST", "PUT", "DELETE", "HEAD"],
+      "AllowedHeaders": ["*"]
+    }
+  ]
+}'
```

Comment:
What purpose do these lines of code do for the localstack?

Reply:
This allows the frontend to download the book in the reader while running on localhost:3000. Without it, the frontend request will be blocked by CORS.